PICMI_Python/particles.py

```diff
@@ -19,6 +19,7 @@ class PICMI_Species(_ClassWithInit):
     Species
       - particle_type=None: A string specifying an elementary particle, atom, or other, as defined in the openPMD 2 species type extension, openPMD-standard/EXT_SpeciesType.md
       - name=None: Name of the species
+      - method: One of 'Boris', 'Vay', 'Higuera-Cary', 'Li' , 'free-streaming', and 'LLRK4' (Landau-Lifschitz radiation reaction formula with RK-4) (string)
```

Comment:
Following the pattern, this should be `method=None: One of ...`

Reply:
Thank you!  I think I made all the changes according to the reviews.  Please give this version a look.